src/field-renderer/FieldRenderer.jsx

```diff
@@ -6,71 +6,109 @@ import { ExpandMore } from '@edx/paragon/icons';
 
 const FormFieldRenderer = (props) => {
   let formField = null;
-  const { fieldData, onChangeHandler, value } = props;
+  const {
+    errorMessage, fieldData, onChangeHandler, isRequired, value,
+  } = props;
+
+  const handleFocus = (e) => {
+    if (props.handleFocus) { props.handleFocus(e); }
+  };
+
+  const handleOnBlur = (e) => {
+    if (props.handleBlur) { props.handleBlur(e); }
+  };
 
   switch (fieldData.type) {
     case 'select': {
       if (!fieldData.options) {
         return null;
       }
       formField = (
-        <Form.Group controlId={fieldData.name} className="mb-3">
+        <Form.Group controlId={fieldData.name}>
           <Form.Control
             as="select"
             name={fieldData.name}
             value={value}
             onChange={(e) => onChangeHandler(e)}
             trailingElement={<Icon src={ExpandMore} />}
             floatingLabel={fieldData.label}
+            onBlur={handleOnBlur}
+            onFocus={handleFocus}
           >
```

Comment:
@attiyaIshaque Please add the `aria-invalid` attribute in all the fields that show error state

Ref: https://github.com/openedx/frontend-app-authn/pull/571/files

Reply:
Added. :) Thanks for pointing out this.